CHANGES.rst

```diff
@@ -2,8 +2,12 @@
  Changes
 =========
 
-5.0 (unreleased)
-================
+4.9.4 (unreleased)
+==================
+
+- Set `IDecimal` attributes `min`, `max` and `default` as `Decimal` type
```

Comment:
Double backticks ("``") for literals, please. Otherwise with a single backtick ("`") you're using the default role, which tends to render in italics unless otherwise configured (as happens when the changelog is rendered on PyPI).